rm-services-rh-stub/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+flask = "==0.10.1"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
3.7 here; 3.6 earlier... does this affect in any way the project? I know that there are not too many changes (definitely not major ones) between 3.6 and 3.7 and 3.7.1. Please mention which one should be here and elsewhere